subjects/social-network/audit/README.md

```diff
@@ -4,16 +4,26 @@
 
 ##### Open the project
 
-###### Is the file system for the backend well organized?
-
-##### Open the project
+###### When examining the file system of the backend, did you find a well-organized structure, similar to the example provided in the subject, with a clear separation of packages and migrations folders?
 
 ###### Is the file system for the frontend well organized?
 
+#### Backend
+
+###### Does the backend include a clear separation of responsibilities among its three major parts - Server, App, and Database?
+
+###### Is there a server component that effectively receives incoming requests and serves as the entry point for all requests to the application?
```

Comment:
`Server component` might sound like a "frontend-related" thing due to the recent evolution of frontend frameworks. I would change it with something like `Is there a clear part of the backend that effectively ....`
![image](https://github.com/01-edu/public/assets/36924670/0cddd7a6-12ad-44ff-b094-07bf2413c640)

Reply:
Done, thanks!